apps/api-server/src/lib/mail.js

```diff
@@ -1,7 +1,7 @@
 const config = require('config');
```

Comment:
Het enige wat er in mail.js is bijgekomen is dat elke send<specifieke>mail functie in een try-catch gewrapped is.